### ISSUE TITLE
Refactor code to use the same internal api for cli and interactive interface

### ISSUE THREAD

**Issue by l-wegner** [deep5050/radio-active#79]
**Is your feature request related to a problem? Please describe.**
Currently there are function, which are available in cli and some are available in interactive interface. Adding new functionality is quite distributed.

**Describe the solution you'd like**
Having an internal API, which encapsulates the player, the recording, the favs, and the stations db, would allow cleaning up the main.py regarding the parsed arguments and allows for implementing a better interactive interface.
Further more extract the ui printing.

In the best case the `utilities.py` is almost empty afterwards.

**Additional context**

**Comment by l-wegner** [comment]
I'll start this tomorrow.

**Comment by deep5050** [comment]
yes, I also feel that. need to refactor before I add new features. we can discuss the design

**Comment by umop3plsdn** [comment]
omg this would be amazing or even the ability to use socat to listen in to the dumped json so we can integrate into other apps like current song or station info

**Comment by deep5050** [comment]
> omg this would be amazing or even the ability to use socat to listen in to the dumped json so we can integrate into other apps like current song or station info

did not get it!
